docs/specs/Broker-CI.md

```diff
@@ -0,0 +1,103 @@
+# Broker CI
+
+The Broker CI spec will outline how the project is going create CI.
+
+## Introduction
+The Broker CI will involve multiple projects: ansible-service-broker
+and apb-examples repo. The goal is to outline the possible CI for
+each repo to achieve a successful CI run outlined in the next section.
+
+## Successful CI Run
+A successful CI run is spawning the broker and using the latest
+rhscl-postgresapb and mediawiki-apb images to build a MediaWiki page.
+
+## Environment
+The expected running service in the environment in all test cases will be:
+- Docker
+- Openshift Cluster
+- Service-Catalog
+
+The services that will be deployed every CI run are:
+- ansible-service-broker
+- mediawiki
+- postgresql
+
+The environment will be deployed using [catasb](https://github.com/fusor/catasb)
+
+## Additional Environment Questions
+When to redeploy catasb?
+Deploy catasb each CI run?
+Redeploy catasb every new service-catalog image?
+Redeploy catasb nightly?
+Should there be an already existing OpenShift Cluster to land pods on?
+Should Jenkins be run in the OpenShift Cluster?
```

Comment:
All good questions. Redeploying catasb each time will always remove any variables and pick up changes to catasb. But it will make the CI run way longer, do we care if it's long?